includes/hook.item-comments.php

```diff
@@ -41,7 +41,7 @@
 }
 
 if (in_array($item, $enabledItems) && $iaCore->get('comments_enabled')) {
-    $iaComment = $iaCore->factoryPlugin('comments', iaCore::FRONT, 'comment');
+    $iaComment = $iaCore->factoryModule('comment',  'comments');
```

Comment:
It's ok to keep as is here, as you might want to use this hook when IA_CURRENT_MODULE is not defined.